keyboards/wilba_tech/zeal60/rules.mk

```
@@ -31,7 +31,7 @@ MIDI_ENABLE = no            # MIDI controls
 AUDIO_ENABLE = no           # Audio output on port C6
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. 
+RGBLIGHT_ENABLE = custom        # Enable WS2812 RGB underlight. 
```

Pretty sure this should be reverted.

---

This definitely should be reverted. 

```suggestion
RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
```

---

Yes, you are correct, sorry about that. Thanks for the patience guys! :) 👍